src/Javascript/index.js

```diff
@@ -11,23 +11,18 @@
 
 import "./web_modules/tocca.js"
 import * as wn from "./web_modules/webnative.js"
+import"./web_modules/webnative-elm.js"
 
 import "./custom.js"
 
 import * as analytics from "./analytics.js"
 import * as fs from "./fs.js"
 import * as ipfs from "./ipfs.js"
-import * as media from "./media.js"
-import { throttle } from "./common.js"
-import { setup } from "./sdk.js"
 
 
 // | (• ◡•)| (❍ᴥ❍ʋ)
 
 
-self.wn = wn
-
-
```

Comment:
Not sure whether I should have actually removed this. What's its use?

Reply:
Just to make it available in the browser console.
(`self = window` in the ui context)